xen/arch/arm/coproc/coproc.c

```diff
@@ -267,6 +267,22 @@ bool_t coproc_is_attached_to_domain(struct domain *d, const char *path)
     return is_created;
 }
 
+/*
+ * To make a decision can this device be assigned to the IOMMU context.
+ * As we only have device-tree support for now treat potential coproc device
+ * as a device-tree device.
+ */
+bool_t coproc_is_protected(device_t *dev)
```

Comment:
function name is useless, imho: is_protected from/for what? Comment says it is here to test if IOMMU context should be used or not. Pls consider renaming appropriately